db/schema.rb

```diff
@@ -312,6 +312,7 @@
     t.text "sub_topic"
     t.string "viaf"
     t.string "gnd"
+    t.string "task"
```

Comment:
Not too sure about the field name. Other ideas?

Reply:
it's difficult to choose one which is not somehow meaningful with ruby (eg 'type' or 'function'), but I'm really open for any suggestions